integrationTests/jdbc/src/test/java/org/apache/camel/forage/jdbc/SingleIT.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.camel.forage.jdbc;
+
+import java.util.List;
+import java.util.Map;
+import org.apache.camel.forage.integration.tests.ForageIntegrationTest;
+import org.apache.camel.forage.integration.tests.IntegrationTestSetupExtension;
+import org.citrusframework.GherkinTestActionRunner;
+import org.citrusframework.TestActionSupport;
+import org.citrusframework.annotations.CitrusResource;
+import org.citrusframework.annotations.CitrusTest;
+import org.citrusframework.junit.jupiter.CitrusSupport;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.DockerImageName;
+
+@CitrusSupport
+@Testcontainers
+@ExtendWith(IntegrationTestSetupExtension.class)
+public class SingleIT implements TestActionSupport, ForageIntegrationTest {
+
+    @Container
+    static PostgreSQLContainer<?> postgres = new PostgreSQLContainer<>(
+                    DockerImageName.parse("mirror.gcr.io/postgres:15.0").asCompatibleSubstituteFor("postgres"))
+            .withExposedPorts(5432)
+            .withUsername("test")
+            .withPassword("test")
+            .withDatabaseName("postgresql")
+            .withInitScript("singleITInitScript.sql");
+
+    /**
+     * resources are copied into work directory by the extension {@link org.apache.camel.forage.integration.tests.IntegrationTestSetupExtension}
+     */
+    @Override
+    public List<String> resources() {
+        return List.of("route.camel.yaml", "forage-datasource-factory.properties");
+    }
+
+    @TestTemplate
+    @CitrusTest(name = "RunIntegration_Jdbc_Single_IT - {0}")
+    public void singleIT(@CitrusResource GherkinTestActionRunner runner) {
+        // running jbang forage run with required resources and required runtime
+        runner.when(camel().jbang()
+                .custom("forage", "run")
+                .processName("route")
+                .addResources(resources())
+                .withArg(System.getProperty(IntegrationTestSetupExtension.RUNTIME_PROPERTY))
+                .withEnvs(Map.of("JDBC_URL", postgres.getJdbcUrl())));
+        // validation of logged message
+        runner.then(camel().jbang()
```

Comment:
Can you provide (if possible) an example that interact with the Camel route? for example, is it possible to execute a `direct:something` route?

@christophd if this is not possible, I think we can add this feature like I did with https://github.com/megacamelus/kaoto-camel-manager/blob/main/src/main/java/org/kaoto/core/CamelOperations.java#L133

Reply:
I think that we can put a `direct:somehing` into a route.yaml and then read the output via the runner.
Here is an [example](https://github.com/citrusframework/citrus/blob/main/endpoints/citrus-camel/src/test/java/org/citrusframework/camel/integration/CamelRouteIT.java#L47) of route test in Citrus.


As you can see, it is possible to send a message too.